CMakeLists.txt

```diff
@@ -177,6 +179,12 @@ endif()
 if(MATERIALX_TEST_RENDER)
     add_definitions(-DMATERIALX_TEST_RENDER)
 endif()
+if (MATERIALX_DYNAMIC_ANALYSIS)
+    add_definitions(-DMATERIALX_DYNAMIC_ANALYSIS)
```

Comment:
Perhaps we should revert the addition of this preprocessor flag, since it's no longer used in the code?